azurerm/internal/services/network/tests/virtual_hub_connection_resource_test.go

```diff
@@ -131,6 +131,25 @@ func TestAccAzureRMVirtualHubConnection_recreateWithSameConnectionName(t *testin
 	})
 }
 
+func TestAccAzureRMVirtualHubConnection_withRoutingConfiguration(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_virtual_hub_connection", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMVirtualHubConnectionDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMVirtualHubConnection_withRoutingConfiguration(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMVirtualHubConnectionExists(data.ResourceName),
+				),
+			},
+			data.ImportStep(),
```

Comment:
we need more tests here, this only just one create case, can we add multiple routes, updates, removing the route etc?

Reply:
updated